### PR TITLE
OSDOCS#8394: Migrated the Networking topic for the Deploying your Application docs

### PR DESCRIPTION
**Pull request by EricPonvelle**
Version(s):
`enterprise-4.15+`

Issue:
**[OSDOCS-8394](https://issues.redhat.com/browse/OSDOCS-8394)**

Link to docs preview:
- **[Tutorial: Networking](https://75318--ocpdocs-pr.netlify.app/openshift-rosa/latest/cloud_experts_tutorials/cloud-experts-deploying-application/cloud-experts-deploying-application-networking.html)**

QE review:
- [ ] QE has approved this change.

Additional information:
Migrated the Networking page in the Deploying your Application tutorial.

### CODE REVIEW

**Comment by lihongan** [comment]
/LGTM
Thank you!

File: cloud_experts_tutorials/cloud-experts-deploying-application/cloud-experts-deploying-application-networking.adoc [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+toc::[]
+
+//rosaworkshop.io content metadata
+//Brought into ROSA product docs 2023-12-14
+
+This tutorial shows how the OSToy app uses intra-cluster networking to separate functions by using microservices and visualize the scaling of pods.
+
+image::deploying-networking-arch.png[OSToy Diagram]
+
+The diagram shows there are at least 2 separate pods, each with its own service.
```
Comment (bmcelvee) [inline cloud_experts_tutorials/cloud-experts-deploying-application/cloud-experts-deploying-application-networking.adoc:16]: Suggestion: 
s/2/two

File: cloud_experts_tutorials/cloud-experts-deploying-application/cloud-experts-deploying-application-networking.adoc [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+You can view your networking configurations in your OSToy application.
+
+.Procedure
+. In the OSToy application, click *Networking* in the left menu. 
+. Review the networking configuration. The right tile titled "Hostname Lookup" illustrates how the service name created for a pod can be used to translate into an internal ClusterIP address.
++
+image::deploying-networking-example.png[OSToy Networking page]
+
+. Enter the name of the microservice created in the right tile ("Hostname Lookup") following the format of `<service_name>.<namespace>.svc.cluster.local`. You can find this service name in the service definition of `ostoy-microservice.yaml` by running the following command:
```
Comment (bmcelvee) [inline cloud_experts_tutorials/cloud-experts-deploying-application/cloud-experts-deploying-application-networking.adoc:32]: The IBM style guide recommends avoiding parentheticals, but I think it's fine here.

**Comment by bmcelvee** [comment]
One suggestion and one comment. LGTM!

**Comment by lihongan** [comment]
LGTM. thanks

**Pull request merged**